packages/components/addon/components/hds/table/index.js

```diff
@@ -48,6 +52,25 @@ export default class HdsTableIndexComponent extends Component {
     return this.args.isStriped ?? true;
   }
 
+  /**
+   * @param density
+   * @type {string}
+   * @default 'medium'
+   * @description Determines the density of the table cells; options are "short", "medium" and "tall". If no density is defined, "medium" is used.
```

Comment:
Comment: "short", "medium" and "tall" sound more like descriptors for `height` vs. `density`. Have we looked at what other design systems use for similarly named props? @jorytindall @MelSumner @heatherlarsen 

Looking at Gmail, they have a `density` setting with values for "default", "comfortable", and "compact".

Other possible descriptors could be:
* high, medium, low
* compact, medium/default, loose?

or other descriptive words more along those lines...

Reply:
@KristinLBradley there's a long Slack thread, I'll ping you in our channel 👍

Reply:
Something along the lines of what @heatherlarsen mentioned in the Slack thread makes sense to me. 

IE: "dense", "default/medium", and "spacious"

Reply:
I wonder if you'd want to file a JIRA ticket and then we can ask Jory to re-visit when he's back from vacation? Otherwise, I'm tempted to leave it as is for now so we have the alignment with the Figma API. WDYT?